### PR TITLE
Add new tag activtiies [SCI-3341]

### PR DESCRIPTION
**Pull request by aignatov-bio**
Jira ticket: [SCI-3341](https://biosistemika.atlassian.net/browse/SCI-3341)

### What was done
Added new activities for tags.
Added tests for new activities.

### CODE REVIEW

File: app/controllers/tags_controller.rb [hunk 1]
```diff
@@ -25,17 +25,9 @@ def create
...
-                subject: my_module,
-                project:
-                  my_module.experiment.project,
-                team: current_team,
-                message_items: {
-                  my_module: my_module.id,
-                  tag: @tag.id
-                })
+
+        log_activity(:create_tag, @tag.project, tag: @tag.id, project: @tag.project.id)
```
Comment (radiokills) [inline app/controllers/tags_controller.rb]: Shouldn't this be outside `if params.include? "my_module_id"'` block?

File: config/initializers/extends.rb [hunk 2]
```diff
@@ -178,7 +178,7 @@ class Extends
     change_task_due_date: 62,
     remove_task_due_date: 63,
     add_task_tag: 64,
-    edit_task_tag: 65,
+    edit_tag: 65,
```
Comment (radiokills) [inline config/initializers/extends.rb]: Will we have any problems with existing data?

File: app/controllers/tags_controller.rb (hunk 1 above)
Comment (aignatov-bio) [reply app/controllers/tags_controller.rb]: Fix

File: config/initializers/extends.rb (hunk 2 above)
Comment (aignatov-bio) [reply config/initializers/extends.rb]: It was create, but never used. And in Activity Excel had flag "not implement". So i rename it along with other tag activities.

**Pull request merged**